### ISSUE TITLE
Add Bumper Support

### ISSUE THREAD

**Issue by Starsurfer78** [Selective031/RazorBoard#1]
If i read the README right, they are no Bumpers yet. It would be nice if you can add it 2 Pins for it. Perhaps 
Micro Switch or Hall Sensors.

**Comment by Selective031** [comment]
No bumber support yet, I wrote some code a while back which I later removed because the motor spike was very good. But of course there should be bumber support.

**Comment by MagnusPer** [comment]
Could not a ultrasonic sensor do the jobb!
DYP-A02YY PWM / UART DC5V Ultrasonic Ranging Sensors Ultrasonic Waterproof Distance Sensor Waterproof Probe Ultrasonic module - UART serial por

**Comment by Starsurfer78** [comment]
Yes but if the Ultrasonic didn't see the Obstacle then the bumper is a safety device
